### PR TITLE
subclass/container: widget in set_focu_child should be Nullable

### PR DESCRIPTION
**Pull request by alatiera**
https://developer.gnome.org/gtk3/stable/GtkContainer.html#gtk-container-set-focus-child

### CODE REVIEW

File: src/subclass/container.rs [hunk 1]
```diff
@@ -177,9 +177,9 @@ unsafe extern "C" fn container_set_focus_child<T: ObjectSubclass>(
     let instance = &*(ptr as *mut T::Instance);
     let imp = instance.get_impl();
     let wrap: Container = from_glib_borrow(ptr);
-    let widget: Widget = from_glib_borrow(wdgtptr);
+    let widget: Option<Widget> = from_glib_borrow(wdgtptr);
 
-    imp.set_focus_child(&wrap, &widget)
+    imp.set_focus_child(&wrap, widget)
```
Comment (sdroege) [inline src/subclass/container.rs]:
```suggestion
    imp.set_focus_child(&wrap, widget.as_ref())
```

We must never pass owned references to a `from_glib_borrow()` return value anywhere

File: src/subclass/container.rs [hunk 2]
```diff
@@ -24,7 +24,7 @@ pub trait ContainerImpl: ContainerImplExt + WidgetImpl + 'static {
         self.parent_check_resize(container)
     }
 
-    fn set_focus_child(&self, container: &Container, widget: &Widget) {
+    fn set_focus_child(&self, container: &Container, widget: Option<Widget>) {
```
Comment (sdroege) [inline src/subclass/container.rs]:
```suggestion
    fn set_focus_child(&self, container: &Container, widget: Option<&Widget>) {
```

And similar for the other places

**Comment by sdroege** [comment]
CC @BrainBlasted

**Comment by sdroege** [comment]
Thanks, looks good to me! @GuillaumeGomez please merge after CI :)

**Comment by sdroege** [comment]
@GuillaumeGomez CI green

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**